Formula/dar.rb

```diff
@@ -27,11 +40,18 @@ class Dar < Formula
   depends_on "upx" => [:build, :optional]
 
   def install
+    if build.stable? && ENV.compiler == :clang && MacOS.version >= :mavericks
+      ENV.libstdcxx
+    elsif build.devel?
+      ENV.O0
+      ENV.deparallelize
```

Comment:
Not building gcc itself! hehe. I mean building this formula *with* gcc. If it's just a Skylake thing, we can keep support for Mavericks and Yosemite via gcc which has the requisite thread_local feature. Even if O0 is required until upstream has a fix, that should be good enough not to outright drop them, but I'm assuming it's not a big deal either way, since the Mavericks downloads were nil and I'm guessing the Yosemite numbers weren't much better.